CHANGES.rst

```diff
@@ -5,7 +5,15 @@ Changelog of lizard-auth-client
 1.14 (unreleased)
 -----------------
 
-- Nothing changed yet.
+- Put the attemp_auto_login function into a middlware.
```

Comment:
"Put" => "Turned"

(Ja, verleden tijd. WOM was het ermee eens dat dat de enige logische en taalkundig juiste vorm is :-) )